java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -223,6 +223,11 @@ public static void assertPartialColumnsAreEqual(HostColumnVectorCore expected, l
             assertArrayEquals(expected.getUTF8(expectedRow), cv.getUTF8(tableRow),
                 "Column " + colName + " Row " + tableRow);
             break;
+          case DECIMAL32:
+          case DECIMAL64:
+            assertEquals(expected.getBigDecimal(expectedRow), cv.getBigDecimal(tableRow),
```

Comment:
We have already verified that the scale and precision match above.  It would probably be fastest to just add DECIMAL32 after line 197 so we then just verify that the underlying int values match, and have DECIMAL64 be after line 210 so ti does the same thing with longs.